src/main/java/com/esri/samples/portal/oauth/README.md

```diff
@@ -1,42 +1,41 @@
-<h1>OAuth</h1>
+<h1>Authenticate with OAuth</h1>
 
-<p>Authenticate with OAuth 2.0 to retrieve the user's profile information.</p> 
+<p>Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).</p>
 
-<p><img src="OAuth.png"/></p>
+<p><img src="authenticate-with-oauth.png"/></p>
 
-<h2>How to use the sample</h2>
+<h2>Use case</h2>
+
+<p>Your app may need to access items that are only shared with authorized users. For example, your organization may host private data layers or feature services that are only accessible to verified users. You may also need to take advantage of premium ArcGIS Online services, such as geocoding or routing services, which require a named user login.</p>
 
-<p>This sample requires you to setup your own app on arcgis.com. See the <a href="https://github.com/Esri/arcgis-runtime-samples-java/wiki/OAuth">wiki</a> for details.</p>
+<h2>How to use the sample</h2>
 
-<p>Enter the details of the application registered on arcgis.com and click sign-in.
-This will open a sign-in dialog. After the credentials are entered correctly, the sample
-will receive an authorization code from the ArcGIS platform. This authorization code will then be used to obtain an
-access token. This access token is used later to access user's profile.</p>
+<p>When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap layer.</p>
```

Comment:
When I run the app, a web map isn't loaded (just a blank canvas). Only when I authenticate successfully does the map and the traffic layer display. Is that intended? If so, this section should be re-ordered to reflect that.

Reply:
Good spot, I've reworded this section.